src/components/BorrowerProfile/LendCta.vue

```diff
@@ -1,107 +1,140 @@
+					},
+					'lg:tw-rounded-t',
+					'lg:tw-px-4',
+				]"
+			>
+				<div
+					:class="[
+						'tw-pt-1',
+						'tw-col-span-12',
+						{
+							'md:tw-pt-0 md:tw-pb-2': !isSticky,
+							'md:tw-col-start-2 md:tw-col-span-10': isSticky,
+						},
+						'lg:tw-col-span-12',
+						'lg:tw-py-1',
+					]"
 				>
-					<option
-						v-for="priceOption in prices"
-						:key="priceOption"
-						:value="priceOption"
+					<p class="tw-text-h3 tw-pt-3 lg:tw-mb-3 tw-hidden lg:tw-inline-block">
+						{{ lgScreenheadline }}
+					</p>
+					<span class="tw-flex tw-pb-1 lg:tw-pb-3">
+						<kv-ui-select
+							v-if="hideShowLendDropdown"
+							id="LoanAmountDropdown"
+							class="tw-pr-2.5 tw--mb-2"
+							v-model="selectedOption"
+						>
+							<option
+								v-for="priceOption in prices"
+								:key="priceOption"
+								:value="priceOption"
+							>
+								${{ priceOption }}
+							</option>
+						</kv-ui-select>
+						<!-- Lend button -->
+						<kv-ui-button
+							v-if="!showAdding && !inBasket"
+							class="tw-inline-flex tw-flex-1"
+							@click="addToBasket"
+						>
+							{{ ctaButtonText }}
+						</kv-ui-button>
+						<!-- Continue to checkout button -->
+						<kv-ui-button
+							v-if="inBasket"
+							class="tw-inline-flex tw-flex-1"
+							:to="'/basket'"
+						>
+							Continue to checkout
+						</kv-ui-button>
+						<!-- Adding to basket button -->
+						<kv-ui-button
+							v-if="showAdding"
+							class="tw-inline-flex tw-flex-1"
+						>
+							Adding to basket...
+						</kv-ui-button>
+					</span>
+					<p
+						v-if="freeCreditWarning"
+						class="tw-text-h4 tw-text-gray-500 tw-inline-block tw-text-center tw-w-full"
 					>
-						${{ priceOption }}
-					</option>
-				</kv-ui-select>
-				<!-- Lend button -->
-				<kv-ui-button
-					v-if="!showAdding && !inBasket"
-					class="tw-inline-flex tw-flex-1"
-					@click="addToBasket"
-				>
-					{{ ctaButtonText }}
-				</kv-ui-button>
-				<!-- Continue to checkout button -->
-				<kv-ui-button
-					v-if="inBasket"
-					class="tw-inline-flex tw-flex-1"
-					:to="'/basket'"
-				>
-					Continue to checkout
-				</kv-ui-button>
-				<!-- Adding to basket button -->
-				<kv-ui-button
-					v-if="showAdding"
-					class="tw-inline-flex tw-flex-1"
-				>
-					Adding to basket...
-				</kv-ui-button>
-			</span>
-			<p
-				v-if="freeCreditWarning"
-				class="tw-text-h4 tw-text-gray-500 tw-inline-block tw-text-center tw-w-full"
-			>
-				Not eligilble for lending credit
-			</p>
-			<p
-				v-if="allSharesReserved"
-				class="tw-text-h4 tw-text-gray-500 tw-inline-block tw-text-center tw-w-full"
+						Not eligilble for lending credit
+					</p>
+					<p
+						v-if="allSharesReserved"
+						class="tw-text-h4 tw-text-gray-500 tw-inline-block tw-text-center tw-w-full"
+					>
+						All shares reserved
+					</p>
+				</div>
+			</kv-grid>
+			<kv-grid
+				:class="[
+					'tw-grid-cols-12 tw-gap-x-2',
```

Comment:
The grid gaps are handled by `kv-grid`, since they change between breakpoints. I think you can remove all of these `tw-gap-x-2` classes across the board